src/components/Main/Team/Profile.tsx

```diff
@@ -9,9 +10,9 @@ interface Props {
 function Profile(props: Props) {
   return (
     <div>
-      <img src="#" alt={`Headshot of ${props.name}`} />
-      <p>{props.name}</p>
-      <p className="p2">{props.position}</p>
+      {props.img==="" ? <p></p> : <img className={isMobile ? "teamImgMob d-block mx-auto" : "teamImg d-block mx-auto"} src={props.img} alt={`Headshot of ${props.name}`} />}
```

Comment:
instead of `{props.img==="" ? <p></p>`
try `{props.img && <img class...`

try `className={`${isMobile ? "teamImgMob" : "teamImg" d-block mx-auto`}` this is called string interpolation